dotnet/src/Agents/OpenAI/OpenAIAssistantChannel.cs

```diff
@@ -8,14 +8,19 @@
 using System.Threading.Tasks;
 using Azure;
 using Azure.AI.OpenAI.Assistants;
+using Microsoft.Extensions.Logging;
 using Microsoft.SemanticKernel.ChatCompletion;
 
 namespace Microsoft.SemanticKernel.Agents.OpenAI;
 
 /// <summary>
 /// A <see cref="AgentChannel"/> specialization for use with <see cref="OpenAIAssistantAgent"/>.
 /// </summary>
-internal sealed class OpenAIAssistantChannel(AssistantsClient client, string threadId, OpenAIAssistantConfiguration.PollingConfiguration pollingConfiguration)
+internal sealed class OpenAIAssistantChannel(
+    AssistantsClient client,
```

Comment:
What happend to the logger on `AgentChannel`?

![image](https://github.com/microsoft/semantic-kernel/assets/66376200/e61b231a-4a0b-4722-821b-14f60c4be4ad)

This feels like something got lost.